exercises/rotational-cipher/src/example/java/RotationalCipher.java

```diff
@@ -0,0 +1,44 @@
+
+public class RotationalCipher {
+
+    int shiftKey;
+
+    RotationalCipher(int shiftKey) {
+        this.shiftKey = shiftKey;
+    }
+
+    public String rotate(String data) {
+        StringBuilder dataStringBuilder = new StringBuilder();
+        for (char c : data.toCharArray()) {
+            if (Character.isUpperCase(c)) {
+                dataStringBuilder.append(getReplacementCharacter(c, 'A', 'Z'));
+            } else if (Character.isLowerCase(c)) {
+                dataStringBuilder.append(getReplacementCharacter(c, 'a', 'z'));
+            } else {
+                dataStringBuilder.append(c);
+            }
+        }
+        return dataStringBuilder.toString();
+    }
+
+    /**
+     * 
+     * @param characterToReplace
+     * @param alphabetCaseStart
+     * @param alphabetCaseEnd
+     * @param shiftKey
+     * @return ReplacementCharacter
```

Comment:
`shiftKey` is no longer a parameter of this method :) It's also not very useful to have `@param` annotations that just repeat the name of the parameter. So I would remove the `@param` and `@return` annotations and just leave the last part of the comment :) It's just a suggestion though, it doesn't matter too much :)